src/System Application/App/Cryptography Management/src/RSAImpl.Codeunit.al

```diff
@@ -0,0 +1,248 @@
+namespace System.Security.Encryption;
+
+using System;
+#if not CLEAN24
+#pragma warning disable AL0432
+codeunit 4 "RSA Impl." implements SignatureAlgorithm, "Signature Algorithm v2"
+#pragma warning restore AL0432
+#else
+codeunit 4 "RSA Impl." implements "Signature Algorithm v2"
+#endif
+{
+    Access = Internal;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+
+    var
+        DotNetRSA: DotNet RSA;
+
+    procedure InitializeRSA(KeySize: Integer)
+    begin
+        DotNetRSA := DotNetRSA.Create(KeySize);
+    end;
+
+    procedure GetInstance(var DotNetAsymmetricAlgorithm: DotNet AsymmetricAlgorithm)
```

Comment:
Is this being used anywhere? I don't think so?
Remove if unused.

Reply:
I had copied this from an existing implementation. But it is not really needed. 
I have removed the function.

Reply:
@MattiLE I'm sorry, I didn't realize it was part of the interface. Please revert the removal. 
Only just saw the build failure :(

Reply:
@darjoo  No problem. Is already back in place. :)